website/content/en/highlights/2023-08-30.md

```diff
@@ -0,0 +1,33 @@
+---
+date: "2023-08-30"
+title: "YAML default configuration format"
+description: "YAML is now the default configuration format"
+authors: ["pront"]
+pr_numbers: [18325, 18345, 18378, 18388, 18420]
+release: "0.33.0"
+hide_on_release_notes: false
+badges:
+  type: announcement
+---
+
+## What is changing?
+
+We are thrilled to announce that [YAML](https://yaml.org/) is now the default Vector configuration format.
+
+YAML is unambiguous, offers great readability due to its clean syntax and is a widely adopted configuration language. In addition, many tools support YAML out of the box.
+
+Existing TOML and JSON configurations are not affected by this decision and will work as usual.
+
+## Action Needed
+
+If you are relying on Vector auto-loading `/etc/vector/vector.toml`, this behavior will change in the next release. The `/etc/vector/vector.yaml` will become the new default path. You can keep using the same config by providing the following option `--config /etc/vector/vector.toml` explicitly. Alternatively, you can convert your existing configuration to YAML and write it to the new default path.
```

Comment:
I think we should mention that it will fallback to `vector.toml` for 0.33.0 but not in 0.34.0.